copy/copy_windows.go

```diff
@@ -37,6 +37,10 @@ func getFileSecurityInfo(name string) (*windows.SID, *windows.ACL, error) {
 }
 
 func (c *copier) copyFileInfo(fi os.FileInfo, src, name string) error {
+	if c.modeSet != nil {
+		return errors.Errorf("non-octal mode not supported on windows")
+	}
```

Comment:
Don't think a module exists for this but I think on Windows we could use `golang.org/x/sys/windows` to set corresponding Windows ACL permissions if we want that in the future (cc @gabriel-samfira)